BinanceExchange.API/Trade/BinTrade.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Threading.Tasks;
+using BinanceExchange.API.Client;
+using BinanceExchange.API.Enums;
+using BinanceExchange.API.Models.Request;
+
+//"dynamic" requires a reference to Microsoft.CSharp
+
+namespace BinanceExchange.API.Client.Trade
+{
+    public class MarginTrade
+    {
+        /// <summary>
+        /// Send isolated account buy command
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="symbol">the trading symbol (BTCUSDT...)</param>
+        /// <param name="asset">for which the buy power is queried</param>
+        /// <returns></returns>
+        public static async Task<decimal> MaxBuyPowerInAsset(BinanceClient client, string symbol, string asset)
+        {
+            dynamic accountInfo = await client.QueryIsolatedMarginAccountInfo(new IsolatedMarginAccountInfoRequest()
+            {
+                Symbols = symbol,
+                TimeStamp = DateTime.Now.Ticks
+            });
+
+            dynamic x = accountInfo.assets[0];
+
+            // query the account free asset coins:
+            decimal freeCoins = 0m;
+            if(asset == ((dynamic)x).baseAsset.asset.Value)
+            {// the requested asset is the base
+                freeCoins = decimal.Parse(((dynamic)x).baseAsset.free.Value);
+            }
+            else if(asset == ((dynamic)x).quoteAsset.asset.Value)
+            {// the requested asset is the one used for quotes
+                freeCoins = decimal.Parse(((dynamic)x).quoteAsset.free.Value);
```

Comment:
Why is it called `freeCoins`?

Reply:
free is like cash, i used their term (.baseAsset.free.Value) - the free amount you may use.
regarding Coins, i'll take any suggestion